src/proto/grpc/testing/services.proto

```diff
@@ -37,13 +37,22 @@ import "src/proto/grpc/testing/control.proto";
 package grpc.testing;
 
 service BenchmarkService {
-  // One request followed by one response.
-  // The server returns the client payload as-is.
+  // One request followed by one response (ping-pong).
+  // Each side has its own message size as set in the configuration
```

Comment:
worth clarifying:
Request and response size is specified by configuration sent by driver to the client. Server doesn't receive any payload size configuration and always responds with response size based on SimpleRequest.response_size of corresponding request.